flake.nix

```diff
@@ -3,6 +3,7 @@
     flakeModule = ./nix/flake-module.nix;
 
     lib = ./nix/lib.nix;
+    evalModules = import ./nix/eval-modules.nix;
```

Comment:
Since `lib` already exists for this purpose, you want to put those two functions inside of it.

Reply:
What bugs me a bit about this is that if we put the functions inside lib, we first need to pass in "lib", and then pass in "pkgs" again which already has lib. This makes using the function a bit strange, but I can do it if you want.

Reply:
You make the arguments like this:

```nix
{ pkgs, lib ? pkgs.lib, ... }
```

Then only one argument (the first) is non-optional.

Reply:
Ok, done. This is a breaking change, though.